profile/README.md

```diff
@@ -1,10 +1,32 @@
-### Run better together with Open Source
+# Welcome to SAP on GitHub
+
+<img src="https://www.sap.com/content/dam/application/shared/logos/sap-logo-svg.svg" alt="SAP Logo" width="200" />
+
+:wave: Welcome to the official GitHub presence of SAP. SAP is the leading enterprise application and business AI company. We stand at the intersection of business and technology, where our innovations are designed to directly address real business challenges and produce real-world impacts.
```

Comment:
The last line `where our innovations .... ` sounds a bit marketing like, of which I am okay, I mean if we don't advertise on our own pages then we kind of do not follow the `do good and talk about it` philosophy . but it would good to get a final say from @christianneu.